tfhe/benches/core_crypto/pbs_bench.rs

```diff
@@ -121,25 +99,36 @@ fn throughput_benchmark_parameters<Scalar: UnsignedInteger>(
 fn multi_bit_benchmark_parameters<Scalar: UnsignedInteger + Default>(
 ) -> Vec<(String, CryptoParametersRecord<Scalar>, LweBskGroupingFactor)> {
     if Scalar::BITS == 64 {
-        vec![
-            PARAM_MULTI_BIT_MESSAGE_1_CARRY_1_GROUP_2_KS_PBS,
-            PARAM_MULTI_BIT_MESSAGE_2_CARRY_2_GROUP_2_KS_PBS,
-            PARAM_MULTI_BIT_MESSAGE_3_CARRY_3_GROUP_2_KS_PBS,
-            PARAM_MULTI_BIT_MESSAGE_1_CARRY_1_GROUP_3_KS_PBS,
-            PARAM_MULTI_BIT_MESSAGE_2_CARRY_2_GROUP_3_KS_PBS,
-            PARAM_MULTI_BIT_MESSAGE_3_CARRY_3_GROUP_3_KS_PBS,
-        ]
-        .iter()
-        .map(|params| {
-            (
-                params.name(),
-                <MultiBitPBSParameters as Into<PBSParameters>>::into(*params)
-                    .to_owned()
-                    .into(),
-                params.grouping_factor,
-            )
-        })
-        .collect()
+        let parameters = if cfg!(feature = "gpu") {
+            vec![
+                PARAM_MULTI_BIT_MESSAGE_1_CARRY_1_GROUP_3_KS_PBS,
+                PARAM_MULTI_BIT_MESSAGE_2_CARRY_2_GROUP_3_KS_PBS,
+                PARAM_MULTI_BIT_MESSAGE_1_CARRY_1_GROUP_4_KS_PBS,
+                // PARAM_MULTI_BIT_MESSAGE_3_CARRY_3_GROUP_3_KS_PBS,
```

Comment:
Do we want to benchmark this parameter set at some point ?

Reply:
Actually yes. Btw @IceTDrinker this PR introduces the GROUP_4 parameter set for MESSAGE_1 CARRY_1, should I keep it in the PR or remove it?

Reply:
no no, no group 4 please

Reply:
sure I'll remove it sorry

Reply:
no worries ! 🙂